lib/nexpose/scan.rb

```diff
@@ -11,6 +11,16 @@ def scan_device(device)
       scan_devices([device])
     end
 
+    # Perform an ad hoc scan of a single device at a specific time.
+    #
+    # @param [Device] device Device to scan.
+    # @param [Array[adhoc_schedules]] list of scheduled times at which to run
+    # @return [Status] whether the request was successful
+    #
+    def scan_device_with_schedule(device, schedule)
```

Comment:
I thought that when I was looking at it to start with. However these are kind of the wrong way round name wise. the asset one actually takes the hostname/ip/range to scan and the device one takes the device(asset) id. they've both been there for starting ad-hoc scans for a while now and I just copied the pattern as it seemed sensible to give options for defining the scan targets since both ways are valid.